### DISCUSSION TITLE
Best way to duplicate Directus

### DISCUSSION THREAD

**Discussion by Silbad** [directus/directus#2832] [Archive]
Hello,

Currently, what is the best way to duplicate Directus (to have two identical dev & prod environments)?
* Export / import schema (I saw that this is planned in a future version)
* Export / import database
* Other ?

I saw : https://docs.directus.io/guides/projects.html#multitenancy but I have to create them on differents servers ;)
And I don't want to recreate everything manually, it's tedious and error-prone.

Thanks

**Comment by rijkvanzanten** [comment]
Option 2 (Export / import database) should work fine

**Reply by Silbad** [reply]
ok thanks, to be sure, the steps :
1 - install directus (prod)
2 - create project / connect database / get super admin password (prod)
3 - export (dev) / import database (prod)
